packages/ui/src/components/RichText/index.module.scss

```diff
@@ -0,0 +1,32 @@
+.utrecht-richtext {
+  .utrecht-heading-1:first-child {
+    --utrecht-heading-1-margin-block-start: 0;
+  }
+  .utrecht-heading-2:first-child {
```

Comment:
Ik denk dat deze manier een probleem veroorzaakt met de spacing, doordat alle margin-block-start van de headings verwijderd worden.

<img width="819" alt="Screenshot 2024-06-28 at 12 14 15" src="https://github.com/frameless/strapi/assets/53454241/020893e0-32e8-4421-b72e-8969dc76dc6c">

Reply:
Ja, deze aanpak werkt dus echt niet in de praktijk.